jupyterhub/spawners.py

```diff
@@ -212,7 +213,10 @@ def get_pod_manifest(self):
             volume for volume in self.volumes if volume["name"] != git_volume_name
         ]
         if not options.get("pvc_name"):
-            volume = {"name": git_volume_name, "emptyDir": {}}
+            volume = {
+                "name": git_volume_name,
+                "emptyDir": {"sizeLimit": server_options["disk_request"]},
+            }
```

Comment:
I think it would make sense to increase the pods limit and the request for ephemeral-storage by the emptyDir size limit here (if they are set). Then the value that is set through the charts would be the same independent of wether we're using a PVC or not (ie the amount of local storage that can be used for the the whole runtime environment).

Reply:
I had to add some code to the spawner to achieve this. But it is done. I tested it - it works.